rust/private/rust.bzl

```diff
@@ -0,0 +1,748 @@
+# Copyright 2015 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
```

Comment:
the old doc was pasted in the readme

Reply:
These docs aren't useful here, are they? I kind of hate how they clutter up the source file, so if they're not doing any good, I vote to remove them (and put them or something like them in a public-visible place).

(FTR: I think the old comments were being used to generate a skydoc page somewhere)